go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/fleetdm/fleet/v4
 go 1.21.7
 
 require (
-	cloud.google.com/go/pubsub v1.33.0
+	cloud.google.com/go/pubsub v1.36.1
```

Comment:
Hmmm not sure why it updated the version of many packages like that, I just added specific new dependencies to parse the file types... If we're uncomfortable with those updates I'll try to revert them, but on the other hand  it's not bad to keep our dependencies up to the latest compatible versions...

Reply:
I tried again from the original `go.mod` and just adding `go get github.com/sassoftware/relic/v7` automatically upgrades those packages, it's probably the version required to work with this dependency:

```
$ go get github.com/sassoftware/relic/v7/lib/comdoc
go: upgraded cloud.google.com/go v0.110.8 => v0.112.0
go: upgraded cloud.google.com/go/compute v1.23.0 => v1.23.4
go: upgraded cloud.google.com/go/iam v1.1.2 => v1.1.6
go: upgraded cloud.google.com/go/kms v1.15.2 => v1.15.6
go: upgraded cloud.google.com/go/pubsub v1.33.0 => v1.36.1
go: upgraded cloud.google.com/go/storage v1.30.1 => v1.36.0
go: upgraded github.com/Azure/azure-sdk-for-go v57.0.0+incompatible => v68.0.0+incompatible
go: upgraded github.com/Azure/go-autorest/autorest v0.11.24 => v0.11.29
go: upgraded github.com/Azure/go-autorest/autorest/adal v0.9.18 => v0.9.23
go: upgraded github.com/Azure/go-autorest/autorest/azure/auth v0.5.8 => v0.5.12
go: upgraded github.com/Azure/go-autorest/autorest/azure/cli v0.4.3 => v0.4.6
go: upgraded github.com/DataDog/zstd v1.4.5 => v1.5.5
go: upgraded github.com/aws/aws-sdk-go-v2 v1.9.1 => v1.24.1
go: upgraded github.com/aws/aws-sdk-go-v2/config v1.7.0 => v1.26.6
go: upgraded github.com/aws/aws-sdk-go-v2/credentials v1.4.0 => v1.16.16
go: upgraded github.com/aws/aws-sdk-go-v2/feature/ec2/imds v1.5.0 => v1.14.11
go: upgraded github.com/aws/aws-sdk-go-v2/internal/ini v1.2.2 => v1.7.3
go: upgraded github.com/aws/aws-sdk-go-v2/service/internal/presigned-url v1.3.0 => v1.10.10
go: upgraded github.com/aws/aws-sdk-go-v2/service/kms v1.5.0 => v1.27.9
go: upgraded github.com/aws/aws-sdk-go-v2/service/sso v1.4.0 => v1.18.7
go: upgraded github.com/aws/aws-sdk-go-v2/service/sts v1.7.0 => v1.26.7
go: upgraded github.com/aws/smithy-go v1.8.0 => v1.19.0
go: upgraded github.com/beevik/etree v1.1.0 => v1.3.0
go: upgraded github.com/cpuguy83/go-md2man/v2 v2.0.2 => v2.0.3
go: upgraded github.com/felixge/httpsnoop v1.0.3 => v1.0.4
go: upgraded github.com/go-logr/logr v1.2.4 => v1.4.1
go: upgraded github.com/golang-jwt/jwt/v4 v4.4.2 => v4.5.0
go: upgraded github.com/golang/glog v1.1.0 => v1.1.2
go: upgraded github.com/google/s2a-go v0.1.4 => v0.1.7
go: upgraded github.com/google/uuid v1.3.0 => v1.6.0
go: upgraded github.com/googleapis/enterprise-certificate-proxy v0.2.4 => v0.3.2
go: upgraded github.com/inconshreveable/mousetrap v1.0.0 => v1.1.0
go: upgraded github.com/klauspost/compress v1.16.5 => v1.17.4
go: upgraded github.com/mattn/go-isatty v0.0.17 => v0.0.19
go: upgraded github.com/opencontainers/image-spec v1.0.3-0.20211202183452-c5a74bcca799 => v1.1.0-rc6
go: upgraded github.com/prometheus/client_golang v1.13.0 => v1.18.0
go: upgraded github.com/prometheus/client_model v0.2.0 => v0.5.0
go: upgraded github.com/prometheus/common v0.37.0 => v0.45.0
go: upgraded github.com/prometheus/procfs v0.8.0 => v0.12.0
go: upgraded github.com/rs/zerolog v1.20.0 => v1.32.0
go: added github.com/sassoftware/relic/v7 v7.6.2
go: upgraded github.com/spf13/cobra v1.5.0 => v1.8.0
go: upgraded github.com/ulikunitz/xz v0.5.10 => v0.5.11
go: upgraded go.opentelemetry.io/otel v1.19.0 => v1.22.0
go: upgraded go.opentelemetry.io/otel/metric v1.19.0 => v1.22.0
go: upgraded go.opentelemetry.io/otel/sdk v1.19.0 => v1.22.0
go: upgraded go.opentelemetry.io/otel/trace v1.19.0 => v1.22.0
go: upgraded golang.org/x/oauth2 v0.12.0 => v0.16.0
go: upgraded golang.org/x/sync v0.3.0 => v0.6.0
go: upgraded golang.org/x/time v0.3.0 => v0.5.0
go: upgraded google.golang.org/api v0.128.0 => v0.161.0
go: upgraded google.golang.org/appengine v1.6.7 => v1.6.8
go: upgraded google.golang.org/genproto v0.0.0-20231002182017-d307bd883b97 => v0.0.0-20240205150955-31a09d347014
go: upgraded google.golang.org/genproto/googleapis/api v0.0.0-20231012201019-e917dd12ba7a => v0.0.0-20240125205218-1f4bbc51befe
go: upgraded google.golang.org/genproto/googleapis/rpc v0.0.0-20231012201019-e917dd12ba7a => v0.0.0-20240125205218-1f4bbc51befe
go: upgraded google.golang.org/grpc v1.58.3 => v1.61.0
go: upgraded howett.net/plist v1.0.0 => v1.0.1
go: upgraded software.sslmate.com/src/go-pkcs12 v0.0.0-20210415151418-c5206de65a78 => v0.4.0
```